registration-processor-default.properties

```diff
@@ -289,7 +289,7 @@ registration.processor.reprocess.limit=500
 registration.processor.pause.packets.for.backpressure=true
 # The reprocessor scheduler configurations
 # The elapse time (in sec) beyond which the rids will be considered for reprocessing
-registration.processor.reprocess.elapse.time=900
+registration.processor.reprocess.elapse.time=7200
```

Comment:
why value is not match this ? https://github.com/mosip/mosip-config/blob/upgrade-v2/sandbox/registration-processor-mz.properties#L377

Reply:
updated sir.